StripePayments/StripePayments/Source/API Bindings/STPAPIClient+LinkAccountSession.swift

```diff
@@ -49,7 +49,27 @@ extension STPAPIClient {
             customerEmailAddress: customerEmailAddress,
             completion: completion
         )
-
+    }
+    
+    func createLinkAccountSessionForDeferredIntent(
+        sessionId: String,
+        onBehalfOf: String?,
+        completion: @escaping STPLinkAccountSessionBlock
+    ) {
+        let endpoint: String = "connections/link_account_sessions_for_deferred_payment"
+        var parameters: [String: Any] = [
+            "unique_id": sessionId,
+        ]
+        if let onBehalfOf {
+            parameters["on_behalf_of"] = onBehalfOf
+        }
```

Comment:
cc @jameswoo-stripe here's what the request looks like for now, until we get clarity on any other parameters we should be sending.

Reply:
Should we be passing verification_method, customer, amount, currency at least?

Reply:
I don't know!  Are you passing any of these?

Reply:
Now passing verification_method, amount, currency.  Not passing customer, since it doesn't work.